docusaurus/src/theme/DocSidebar.js

```diff
@@ -8,15 +8,22 @@ const WEB_LINKS = [
   [<>© Stream.IO, Inc. <br/> All Rights Reserved.</>, '/']
 ]
 
+const addTitle = (sidebarItems) =>
+  sidebarItems.map(({ label, items, ...props }) => ({ 
+    ...props,
+    ...(items && { items: addTitle(items) }),
+    label: <span title={label}>{label}</span>
```

Comment:
why is this needed?

Reply:
To add the `title` property to links so browsers will show up tooltip when hovering on those links.
The default `DocSidebar` doesn't allow custom props for items **but** it let us use html inside labels